package.json

```diff
@@ -10,9 +10,9 @@
     "type": "git",
     "url": "git+https://github.com/kainino0x/cts-experiment.git"
   },
-  "author": "Kai Ninomiya <kainino@chromium.org>",
+  "author": "WebGPU CTS Contributors",
   "private": true,
-  "license": "W3C 3-clause BSD License",
```

Comment:
Why not W3C 3-clause BSD?

Reply:
`BSD-3-Clause` is a standard (SPDX) string, so using it here allows machine understanding of the field.

AFAICT (from gpuweb/admin), the license that was agreed on for gpuweb is actually the vanilla BSD-3-Clause, not the W3C 3-clause BSD License. I checked that the LICENSE in gpuweb/admin is the SPDX BSD-3-Clause exactly.